src/Tools/ExternalAccess/Razor/Microsoft.CodeAnalysis.ExternalAccess.Razor.csproj

```diff
@@ -46,6 +46,16 @@
     <ProjectReference Include="..\..\..\Workspaces\Remote\Core\Microsoft.CodeAnalysis.Remote.Workspaces.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <!-- No warn on NU1701 until the LSP client targets netstandard2.0 https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1369985/ -->
```

Comment:
This was adapted from[ EditorFeatures.csproj](https://github.com/dotnet/roslyn/blob/bc6ea4d713630210b1a734547234c2871bd2bbc9/src/EditorFeatures/Core/Microsoft.CodeAnalysis.EditorFeatures.csproj#L40) in order to include the VS LSP client package reference, which is needed for access to the `ILanguageClientMiddleLayer` type.

@dibarbet it looks like the underlying issue here was marked as "completed" but I'm still seeing the same errors mentioned in the internal issue. Would you have any context here?

Reply:
@allisonchou depending on when the bug was completed, we may need to depend on an updated version of the VS language client
https://github.com/dotnet/roslyn/blob/main/eng/Versions.props#L155